CHANGELOG.md

```diff
@@ -94,6 +94,7 @@
 - Adds support for pipe delimiter for CSV #3115 (@pratik-m)
 - Add syntax mapping for `/etc/pacman.conf` #2961 (@cyqsimon)
 - Associate `uv.lock` with `TOML` syntax, see #3132 (@fepegar)
+- Add support for [Idris 2 programming language](https://www.idris-lang.org/) #3150 (@buzden)
```

Comment:
Can I ask you to move this up to the relevant part of the "unreleased" section of the changelog please? :slightly_smiling_face:

Reply:
Sure!